### PR TITLE
Only replace config if it doesn't exist. If it does, replace value.

### PR DESCRIPTION
**Pull request by Soren90**
Maybe something like this can solve the issue of server.cfg and autoexec.cfg being replaced each restart?

### CODE REVIEW

File: containerfs/start.sh [hunk 1]
```diff
@@ -64,6 +73,11 @@ writeid
 writeip
 SERVERCFG
 
+else
+sed -i -n 's/^tv_enabe.*/tv_enable $TV_ENABLE/' $CSGO_DIR/csgo/cfg/server.cfg
```
Comment (kaimallea) [inline containerfs/start.sh:77]: typo in `tv_enabe`

**Comment by kaimallea** [comment]
Thanks, looks good! One minor fix is needed

**Comment by Soren90** [comment]
All right. I've fixed the typo now.

**Review by kaimallea — APPROVED** [review]
Thanks, sorry for the delay!

**Pull request merged**